application/helpers/export_helper.php

```diff
@@ -16,12 +16,15 @@
 * Strips html tags and replaces new lines
 *
 * @param $string
+* @param $removeOther   if 'true', removes '-oth-' from the string.
 * @return string
 */
-function stripTagsFull($string)
+function stripTagsFull($string, $removeOther = true)
```

Comment:
Would it be possible to use function composition instead of a new boolean argument? E.g.

stripTagsFull(remoteOther($theThing));

Reply:
`     $string = str_replace('-oth', '', $string);// Why ?`

Why indeed? :laughing: Do we need to test with this removed?

Reply:
> Would it be possible to use function composition instead of a new boolean argument? 

Not sure. I believe not. The "removal" happens after flatennig the text, is not done initially.
Also, we did it like that as to introduce the minimum change on a function which, if remmeber correctly, is used widely.
At last, it is like this on master as well, so I would say is better to keep it like this, at least for this one.

Reply:
> ` $string = str_replace('-oth', '', $string);// Why ?`
> 
> Why indeed?  Do we need to test with this removed?

Just to highlight, the "why" comment was removed.

> Do we need to test with this removed?

I think they are used for specific export functions.
If messing with that, we should test, identify errors that should appear and fix them.

Reply:
Urgh. Does anyone knows the correct behaviour for "-oth-" here?